tools/mason/MasonPublish.chpl

```diff
@@ -459,6 +463,17 @@ proc check(username : string, path : string, trueIfLocal : bool, ci : bool) thro
     writeln(spacer);
   }
 
+  if package {
+    writeln('Checking for fields in manifest file:');
```

Comment:
We could mention the expected version here: `v<package version that we parsed from TOML file>`

Reply:
@ben-albrecht I am unable to understand why should we mention version here. What would be its importance in a missing fields check ?

Reply:
Oops, I commented in the wrong place. I am suggesting this for the case where the git tag version formatting is wrong. Basically we want to communicate what was expected and what we got, so that it's crystal clear to the user what they need to fix.

Reply:
Okay, cool !